src/pages/edge/adobe-journey-optimizer/public-classes/tabs/messaging-edge-event-type.md

```diff
@@ -21,7 +21,9 @@ public enum MessagingEdgeEventType {
     DISMISS(6),
     INTERACT(7),
     TRIGGER(8),
-    DISPLAY(9);
+    DISPLAY(9),
+    DISQUALIFY(10),
```

Comment:
same comment as above, the disqualify event type is not yet available on android messaging

Reply:
we'll wait until android release tomorrow to release these

Reply:
i've updated my pending android messaging pr to include the disqualify event